sw/device/silicon_owner/bl0_tests/BUILD

```diff
@@ -0,0 +1,47 @@
+# Copyright lowRISC contributors.
```

Comment:
Wasn't sure where to put this, but wanted to demo how to build a OTTF-launched test at the BL0 stage. Perhaps in the future we may create a separate Bazel macro like `opentitan_bl0_test` that builds a test at the BL0 stage, and always pulling in the ROM_EXT.

Reply:
`bare_metal` directory is our current example (more of a scaffolding) for a program that runs at the BL0 stage. Could these go to that folder?

Reply:
done.